packages/xo-server/src/xapi/index.js

```diff
@@ -2348,4 +2348,31 @@ export default class Xapi extends XapiBase {
       )
     }
   }
+
+  async isHyperthreading(hostId) {
+    try {
+      const hyperthreading = await this.call(
+        'host.call_plugin',
+        this.getObject(hostId).$ref,
+        'hyperthreading.py',
+        'get_hyperthreading',
+        {}
+      )
+      if (hyperthreading === 'false') {
+        return false
+      }
+      if (hyperthreading === 'true') {
+        return true
+      }
+    } catch (error) {
+      if (
+        error.code === 'XENAPI_MISSING_PLUGIN' ||
+        error.code === 'UNKNOWN_XENAPI_PLUGIN_FUNCTION'
+      ) {
```

Comment:
Why ignore these errors?

Reply:
Here's the convention:
- When `get_hyperthreading` return `true` we display -> `Hyperthreading enabled`
- When `get_hyperthreading` return `false` we display -> `Hyperthreading disabled`
- When `get_hyperthreading` throws `XENAPI_MISSING_PLUGIN` error we display -> `HT detection is only available on XCP-ng`